lib/jira/commands/assign.rb

```diff
@@ -2,8 +2,9 @@ module Jira
   class CLI < Thor
 
     desc "assign", "Assign a ticket to a user"
+    method_option :assignee, aliases: "-a", type: :string, default: nil, lazy_default: "auto", banner: "ASSIGNEE"
```

Comment:
It is pretty tedious to write out the long option which is why I added an alias. The help menu shows both options to the user. I do not plan on using the cli in scripts but rather have a simple one line command.